vulture/core.py

```diff
@@ -415,6 +420,33 @@ def visit(self, node):
             visitor(node)
         return self.generic_visit(node)
 
+    def _handle_ast_list(self, ast_list, parent_node):
```

Comment:
Oh! yes, I was experimenting with some thing, and this change also got committed accidentally.